tests/integration/defs/test_e2e.py

```
@@ -556,7 +560,43 @@ def run_bench(self):
+            'latency': 0
+        }
+
+        lines = output.split('\n')
+        for line in lines:
+            line = line.strip()
+            if 'total token throughput' in line.lower(
+            ) and 'tokens/sec' in line.lower():
+                try:
+                    throughput = line.split(":")[1].strip()
+                    result['throughput'] = throughput
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)